test/rectangle.spec.js

```diff
@@ -178,4 +178,26 @@ describe('Rectangle', () => {
       expect(rect.bottom).to.equal(210);
     });
   });
+  describe('distance from point', () => {
+    let rect;
+    before(() => {
+      rect = new Rectangle(5, 5, 10, 10);
+    });
+    it('calculates distance to point 1', () => {
```

Comment:
Could we give these tests more meaningful names? "...to point inside" "...to point on edge", etc.

Reply:
Will do 👍🏻